release-notes/release-notes/5.8-release-notes.md

```diff
@@ -17,12 +17,28 @@ For instructions on how to upgrade your platform to the latest version, please r
 
 ### Platform
 
+### Updated POST endpoint for Saved Queries
+
+The POST endpoint for Saved Queries now automatically serializes hierarchical JSON to text when the content-type is `application/json`. This update now supports values that are objects or arrays. This update with complex data structures like objects or arrays. This eliminates the need for manual serialization and makes it easier for developers to work with Saved Queries.
+
+
 #### Extract header values in REST API responses
 
 You can now pull specific data from REST API response headers using .NET regex capture groups. This feature gives you more control and flexibility in collecting the data you need when targeting REST API destinations. For more information, see the [REST API destination](../../data-syncs/supported-data-sync-destinations/rest-api.md) page.
 
 ![Extract variables from response header](../../.gitbook/assets/DataSyncs/variable-response-header.png)
 
+### Enhancements to Listener Config Status and Error Handling in data syncs
+
```

Comment:
CIN-06076. I also need to update the Listener Config documentation for this change.

Reply:
Removed. 5.9 candidate.